test/components/ColorPicker.test.js

```diff
@@ -47,15 +47,15 @@ describe('<ColorPicker />', () => {
       expect(wrapper.find('input').prop('value')).toEqual(initialColor.toUpperCase());
     });
 
-    it('should call onChange once value is changed', () => {
-      wrapper.find('input').simulate('change', { target: { value: changedColor }});
-      expect(onChange).toHaveBeenCalled();
-    });
-
-    it('should change input value after change', () => {
-      wrapper.find('input').simulate('change', { target: { value: changedColor }});
-      expect(wrapper.find('input').prop('value')).toEqual(changedColor.toUpperCase());
-    });
+    // it('should call onChange once value is changed', () => {
```

Comment:
Are you planing to restore that tests in the future ?

Reply:
all tests will be re-written once final shape of theme builder is known. I'll create separate issue for tests

Reply:
👍